src/DocRaptor.php

```diff
@@ -58,52 +79,89 @@ public function setDocumentType($document_type)
         return $this;
     }
 
+    /**
+     * @param string $name
+     * @return $this
+     */
     public function setName($name)
     {
         $this->name = $name;
         return $this;
     }
 
+    /**
+     * @param bool $test
+     * @return $this
+     */
     public function setTest($test = false)
     {
         $this->test = (bool)$test;
         return $this;
     }
 
-    # values should be one of 'html' or 'none'
-    #   'none' - (default) skip input validation on the DR side
-    #   'html' - perform validation on the input html on the DR side and
-    #            error if non-parseable input provided
+    /**
+     * Toggle validation of HTML by DocRaptor
+     *
+     * @todo should probably just be a bool flag
```

Comment:
Could probably make it smart based on the doc type: https://docraptor.com/documentation#api_strict